WordPress/src/main/res/drawable/media_settings_gradient_scrim.xml

```diff
@@ -3,7 +3,7 @@
     <gradient
         android:angle="90"
         android:centerY="75%"
-        android:endColor="@color/black_translucent_50"
+        android:endColor="#602e4453"
```

Comment:
Nice, it's much better now. Just wondering where the new color come from? Isn't there a light grey already defined in the prj?

Reply:
`grey_dark_translucent_50` seems very close to your proposed color.

Reply:
OK, that works for me. I made the change in 0ea5229.